CordovaLib/Classes/Private/Plugins/CDVWebViewEngine/CDVWebViewEngine.m

```diff
@@ -383,7 +383,10 @@ - (void)updateSettings:(NSDictionary*)settings
     // prevent webView from bouncing
     if (!bounceAllowed) {
         if ([wkWebView respondsToSelector:@selector(scrollView)]) {
-            ((UIScrollView*)[wkWebView scrollView]).bounces = NO;
+            UIScrollView* scrollView = [wkWebView scrollView];
+            scrollView.bounces = NO;
+            scrollView.alwaysBounceVertical = NO;     /* iOS 16 workaround */
+            scrollView.alwaysBounceHorizontal = NO;   /* iOS 16 workaround */
```

Comment:
wonder if its worth putting these two in
```
if (@available(iOS 16, *)) {
  ...
}
```
seems fine on my ios 15.6.1 phone so probably fine without it too

Reply:
yeah, I think I'm just going to do it. I think it's fine, my bigger concern was availablity of the api, but it's been around since iOS 2.